### DISCUSSION TITLE
remove initTE

### DISCUSSION THREAD

**Discussion by yinvoke** [mdbootstrap/TW-Elements#1989] [Support from community]
In the React version, the initTE operation is too cumbersome. Can it be handled by default instead of being declared in the code😁

**Comment by juujisai** [comment]
Hi! What problems are you experiencing with the `initTE` method? We encourage to use the `initTE` method inside the `useEffect` hook. You can also initialize most components via JS, for example: `new Timepicker(...)`.

Other than that, we also have a React version of the TW-Elements that will be soon extended by more components:
https://tailwind-elements.com/docs/react/
